test/services/onlinetesting/Phase2TestServiceSpec.scala

```diff
@@ -110,7 +110,7 @@ class Phase2TestServiceSpec extends PlaySpec with MockitoSugar with ScalaFutures
       phase2TestService.verifyDataStoreEvents(1, "OnlineExerciseResultSent")
     }
 
-    "exclude adjsutments candidates from the invite" in new Phase2TestServiceFixture {
+    "exclude adjsutments candidates from the invite" ignore new Phase2TestServiceFixture {
```

Comment:
filtering logic in now in Phase2TestMongoRepo not in the service. And test has been added for the same. I can remove this test instead of ignoring.